### DISCUSSION TITLE
can supply some examples using stock or futures?

### DISCUSSION THREAD

**Discussion by quant2008** [nautechsystems/nautilus_trader#1084] [General]
seems current exampes are all about fx. Could supply some examples using stocks or commodity futures?

**Comment by cjdsellers** [comment]
Hi @quant2008 

Thanks for your interest.

So because of the way Nautilus normalizes instruments - backtesting and trading across different asset classes is basically the same. Obviously the data will be different and there are different considerations for futures with expiries / settlement etc - which is not currently built in.

We'll endeavor to provide equities and futures examples eventually in the future.

**Comment by limx0** [comment]
https://github.com/nautechsystems/nautilus_trader/blob/master/examples/live/interactive_brokers_example.py
